app/javascript/packages/document-capture/components/document-capture.jsx

```diff
@@ -51,12 +61,14 @@ function DocumentCapture({ isLivenessEnabled = true }) {
       title: t('doc_auth.headings.document_capture'),
       component: DocumentsStep,
       validate: validateDocumentsStep,
+      fields: ['front', 'back'],
```

Comment:
It seems weird to me to specify the fields "on the outside" like this? Like my expectation is that `DocumentsStep.fields` could tell us the same info?

Reply:
> It seems weird to me to specify the fields "on the outside" like this? Like my expectation is that `DocumentsStep.fields` could tell us the same info?

Hm, yeah, I see your point. In earlier iterations, this had actually been even worse in creating a separate constant `FIELDS_STEP` mapping at the top of the file, and I'd decided to move it here to be closer in line with the definition of the step itself. I'm not sure if you're proposing a new property of the step component (since it doesn't exist), but yeah, I could see something similar to as we're using `validate`. For consistency's sake, maybe an exported member just like `validate`. One of the upsides is that, at least in the case of the documents step, we can simply export `DOCUMENT_SIDES`:

https://github.com/18F/identity-idp/blob/8826d56e6df92f6819da3d96846be2ef366ab152/app/javascript/packages/document-capture/components/documents-step.jsx#L25

In an ideal implementation, I might like it such that the fields might never need to be explicitly listed like this, but instead `FormSteps` could track the step at which `onChange` was called with a particular field name. It's a bit harder to implement, and may not really align with this model of "initial errors", since... if it's initial, then `FormSteps` shouldn't really know at that point where the field would reside. It just so happens in our use-case that we're using it as a submission failure recovery.

Reply:
> For consistency's sake, maybe an exported member just like `validate`. 

Updated in 4bc282215.